bin/collator/src/command.rs

```diff
@@ -333,7 +401,20 @@ pub fn run() -> Result<()> {
         }
         Some(Subcommand::Revert(cmd)) => {
             let runner = cli.create_runner(cmd)?;
-            if runner.config().chain_spec.is_shiden() {
+            if runner.config().chain_spec.is_astar() {
```

Comment:
This part is repeated a lot...perhaps we can look into a helper method or macro in the future.